src/synaptic_sampling_rewardgradient_connection.h

```diff
@@ -586,18 +596,29 @@ ConnectionDataLogger< SynapticSamplingRewardGradientConnection<targetidentifierT
 //
 
 /**
+ * Check syn_spec dictionary for parameters that are not allowed for this
+ * connection. Will issue warning or throw error if a parameter is found.
+ */
+template <typename targetidentifierT>
+void SynapticSamplingRewardGradientConnection<targetidentifierT>::check_synapse_params(const DictionaryDatum& syn_spec)
+    const
+{
+    // FIXME!! Check synaptic parameters here!
```

Comment:
Is this a proper TODO? Do we need an issue to keep track of this on the long term?